src/main/groovy/org/shipkit/internal/gradle/ShipkitJavaPlugin.java

```diff
@@ -20,6 +19,12 @@
  *     <li>{@link BintrayReleasePlugin}</li>
  *     <li>{@link TravisPlugin}</li>
  * </ul>
+ *
+ * Adds behavior:
+ * <ul>
+ *     <li>Applies {@link JavaBintrayPlugin} to all Java projects in a multi-project Gradle build
+ *          (all projects that use Gradle's "java" plugin).</li>
```

Comment:
[`java-library`](https://docs.gradle.org/current/userguide/java_library_plugin.html) plugin would be also covered by that?